java/maven/src/org/netbeans/modules/maven/runjar/LaunchArgPrereqsChecker.java

```diff
@@ -122,6 +123,15 @@ public boolean checkRunConfig(RunConfig config, ExecutionContext con) {
             config.setProperty(MavenExecuteUtils.RUN_APP_PARAMS, 
                     MavenExecuteUtils.joinParameters(appArgsValue));
         }
+        File workingDirectory = injectParams.getWorkingDirectory();
+        if (workingDirectory != null) {
```

Comment:
Note - possibly a different behaviour for missing `cwd` than in N-I debugging (unspecified vs. userdir)

Reply:
Right. When it's not specified, it's not set.